ui-admin/src/study/kits/KitCollection.tsx

```diff
@@ -0,0 +1,165 @@
+                setEnrolleeCodeError(error)
+                setShowEnrolleeCodeScanner(false)
+                setEnrollee(undefined)
+              }}
+              onSuccess={result => {
+                loadEnrollee(result.rawValue)
+                setEnrolleeCodeError(undefined)
+                setShowEnrolleeCodeScanner(false)
+              }}/>
+        }
+        { enrolleeCodeError &&
+            <div className="text-danger">{enrolleeCodeError}</div>
+        }
+      </KitCollectionStep>
+
+      <KitCollectionStep
+        title={'Step 2'}
+        status={isEnrolleeIdentityConfirmed && enrollee ? 'COMPLETE' : 'INCOMPLETE'}
+      >
+        <div className="mb-3">Confirm the enrollee&apos;s identity</div>
+        <TextInput
+          label={'Enrollee Name'}
+          className="mb-2" disabled={true}
+          value={enrollee ? `${enrollee.profile.givenName} ${enrollee.profile.familyName}` : ''}>
+        </TextInput>
+        <TextInput
+          label={'Date of Birth'}
+          className="mb-3" disabled={true}
+          value={enrollee ? dateToDefaultString(enrollee.profile.birthDate) : ''}>
+        </TextInput>
+        <Button className="mb-2" variant={'primary'}
+          disabled={!enrollee} onClick={() => setIsEnrolleeIdentityConfirmed(true)}>
+            Mark as confirmed
+        </Button>
+      </KitCollectionStep>
+
+      <KitCollectionStep
+        title={'Step 3'}
+        status={kitId ? 'COMPLETE' : 'INCOMPLETE'}
+      >
+        <div className="mb-3">Click to open the camera and scan the kit barcode</div>
+        <Button className="mb-2" variant={'primary'} disabled={!isEnrolleeIdentityConfirmed}
+          tooltip={!isEnrolleeIdentityConfirmed ? 'You must complete the prior steps first' : ''}
+          onClick={() => setShowKitScanner(!showKitScanner)}>
+            Click to scan kit barcode
+        </Button>
+        { showKitScanner &&
+            <BarcodeScanner
+              expectedFormats={['code_128']}
+              onError={error => setKitCodeError(error)}
+              onSuccess={result => {
+                setKitId(result.rawValue)
+                setShowKitScanner(false)
+              }}/>
+        }
+        <TextInput
+          className="my-2"
+          disabled={true}
+          value={kitId}>
+        </TextInput>
+        { kitCodeError &&
+            <div className="text-danger">{kitCodeError}</div>
+        }
+      </KitCollectionStep>
+      <div className="d-flex justify-content-end">
+        <Button disabled={!(kitId && enrollee)} variant={'primary'}
+          onClick={() => {
+            setEnrollee(undefined)
+            setIsEnrolleeIdentityConfirmed(false)
+            setKitId(undefined)
+          }}>
+            Submit kit
+        </Button>
+      </div>
+    </div> :
+    <div className="m-2">
+      <div className="alert alert-danger" role="alert">
+          You do not have permission to access this page.
+      </div>
+    </div>
```

Comment:
Minor since this will change eventually, but this ternary is very long - I think it would be more readable to do:

```
if (!user.superuser) {
   return <you do not have permission>
}

return <normal stuff>
```